proposals/incubation/kyverno.md

```diff
@@ -0,0 +1,160 @@
+# Kyverno for incubating 
+
+## Table of Contents
+
+- [Background](#background)
+- [About Kyverno](#about-kyverno)
+- [Progress Since Joining CNCF](#progress-since-sandbox)
+- [Incubation Stage Requirements](#incubation-stage-requirements)
+- [Statement on alignment with the CNCF mission](#statement-on-alignment-with-the-cncf-mission)
+- [Comparison with other CNCF Policy Management projects](#comparison-with-other-cncf-chaos-engineering-projects)
+- [Roadmap](#roadmap)
+
+## Background
+
+Kubernetes has a declarative configuration management system that allows users to specify the desired state of resources which Kubernetes controllers continuously reconcile with the current system state. Kyverno uses Kubernetes custom resource definitions for cluster-wide and namespaced policies, and the [Kubernetes Policy WG policy report](https://github.com/kubernetes-sigs/wg-policy-prototypes/tree/master/policy-report) for reporting policy results.
+
+Kyverno was accepted as a CNCF Sandbox project on [November 9th, 2020](https://docs.google.com/spreadsheets/d/1Nnh_usr0tSZxaUpxTusqeIqKxMmvuEViRkyO9e_Do40/edit#gid=1136111842).
+
+## About Kyverno
+
+Kyverno is a policy engine designed for Kubernetes. With Kyverno, policies are managed as Kubernetes resources and no new language is required to write policies. This allows using familiar tools such as `kubectl`, `git`, and `kustomize` to manage policies. Kyverno policies can validate, mutate, and generate Kubernetes resources. The Kyverno CLI can be used to test policies and validate resources as part of a CI/CD pipeline.
```

Comment:
"Kyverno policies can validate, mutate, and generate Kubernetes resources in addition to verifying container image supply chain security via signing verification and attestations."